src/test/kotlin/interactor/GetCityHasCheapestInternetConnectionInteractorTest.kt

```diff
@@ -19,7 +19,7 @@ internal class GetCityHasCheapestInternetConnectionInteractorTest {
             .validSalaryAndInternetPrices)
         val expectedCity = CheapestInternetFakeData.validSalaryAndInternetPrices.getAllCitiesData()[9]
         // when check for the cheapest city in the list
-        val actual = cityHasCheapestInternet.execute()
+        val actual = cityHasCheapestInternet.invoke()
```

Comment:
replace cityHasCheapestInternet.invoke()
with cityHasCheapestInternet()

Reply:
same in all

Reply:
done